packages/engine/Makefile.toml

```diff
@@ -24,7 +24,7 @@ args = ["build", "-p", "memory", "--profile", "${CARGO_MAKE_CARGO_PROFILE}"]
 condition = { env_true = ["CARGO_MAKE_CI"] }
 private = true
 command = "bash"
-args = ["lib/execution/src/runner/python/setup.sh", "python3.7"]
```

Comment:
I can't comment this many lines above but I think you should remove `features build-nng` from above

Reply:
I think generally the build-nng feature should simply be removed